common/emu_constants.cpp

```diff
@@ -288,15 +290,16 @@ const std::map<bodyType, std::string>& EQ::constants::GetBodyTypeMap()
 		{ BT_InvisMan, "Invisible Man" },
 		{ BT_Special, "Special" },
 	};
+
 	return bodytype_map;
 }
 
 std::string EQ::constants::GetBodyTypeName(bodyType bodytype_id)
 {
-	auto bodytypes = EQ::constants::GetBodyTypeMap();
-	if (!bodytypes[bodytype_id].empty()) {
-		return bodytypes[bodytype_id];
+	if (!EQ::constants::GetBodyTypeMap().find(bodytype_id)->second.empty()) {
+		return EQ::constants::GetBodyTypeMap().find(bodytype_id)->second;
```

Comment:
Can just return the result instead of checking if it's empty since it returns an empty string in the fall through case anyway. If there's worry that the function will be called with an invalid bodyType it should check something like `find(bodytype) != map.end()` instead